landscape/lib/network.py

```diff
@@ -178,7 +189,7 @@ def get_active_device_info(skipped_interfaces=("lo",),
             if netifaces.AF_INET in ip_addresses or extended:
                 results.append(interface_info)
     finally:
-        del sock
+        sock.close()
```

Comment:
Might as well drop that try block and use contextmanagers:
```
with socket.socket():
   ...
```

Reply:
I'm just gonna leave this alone as there seem to be some doubts whether the context manager is supported for sockets in versions of Python earlier than 3.5 https://stackoverflow.com/questions/16772465/how-to-use-socket-in-python-as-a-context-manager